app/src/main/java/org/mozilla/focus/web/WebViewProvider.kt

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.web
+
+import android.content.Context
+import android.preference.PreferenceManager
+import android.util.AttributeSet
+import android.view.View
+import android.webkit.WebSettings
+import org.mozilla.focus.webview.SystemWebView
+
+const val ENGINE_PREF_STRING_KEY = "renderer_preference"
+
+object WebViewProvider : IWebViewProvider {
```

Comment:
\[detekt\] INFO: \[complexity/TooManyFunctions\] Classes with many functions tend to do too many things and often come in conjunction with large classes and can quickly become God classes\. Consider extracting methods to \(new\) classes better matching their responsibility\.